components/ReferenceList/ReferenceList.module.css

```diff
@@ -0,0 +1,34 @@
+.container {
+  background-color: var(--primary-component-color);
+  border-radius: 10px;
+  margin: 10%;
+  padding: 2%;
+  height: 35vh;
```

Comment:
It's not recommended to set height or width relative to the viewport for components. Components should be usable in different context, thus, it's up to the parent container to limit the size.
Same with the `margin`.